tests/assembly/targets/aarch64_be_unknown_netbsd.rs

```diff
@@ -0,0 +1,17 @@
+// assembly-output: emit-asm
+// compile-flags: --target aarch64_be-unknown-netbsd
+// needs-llvm-components: aarch64
```

Comment:
How identical are these tests? Does the `assembly` test suite support revisions? Would it be worth having a single test with a revision for each target, instead of tons of files?